frontend/src/layout/lemonade/TopBar/SitePopover.tsx

```diff
@@ -37,7 +38,7 @@ function AccountInfo(): JSX.Element {
         <div className="AccountInfo">
             <ProfilePicture name={user?.first_name} email={user?.email} size="xl" />
             <div className="AccountInfo__identification SitePopover__main-info">
-                <strong>{user?.first_name}</strong>
+                <b>{user?.first_name}</b>
```

Comment:
[Why not](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#usage_notes) `strong`?

Reply:
👀 Interesting, I had a vague feeling that there was a difference but I wasn't sure. Went for `strong`.